<!DOCTYPE html>

<html>

  <head>
    
    
<!--
        To Customize (1) # of main topics (2) # of subheading topics per main topic, must make this javascript changes:
        
        -(Y) click to scroll to: main topic bar
        -(Y) click to scroll to: subheading bars
        
        -(Y) fix then scroll for main topic bar
        -(Y) fix then scroll for subheading bars
        
        -(Y) scroll to change opacity for main topic bar
        -(Y) scroll to change opacity for subheading bars
        
        Parameters to change for modifying # of topics/subsections
        
        #Topics:
          - number of sec# divs under scrollBar div
          - the actual number of main sections (topicHeader#, topicDescription#, relativeBox#)
        
        #subHeadings for given topic:
          - number of sub## divs under subHeadingScroll# div
          - number of subDes## divs at bottom of relativeBox# div
          
        
    -->
    
    
    <script type="text/javascript">
      window.onload = function() {
        
        
    //scoping function
  (function() {
    
      
      //allows you to flip between this study page and the question page, even if you click within this iframe
  document.body.onkeydown = function(e){
    if(e.keyCode === 191){
      if(window.parent.document.body.scrollTop == 0){
        window.parent.scrollTo(0,1000);
      }
      else{
        window.parent.scrollTo(0,0);
      }
    }
  }
      
        
        var numTopics = document.getElementById('scrollBar').childElementCount;
        var loop = [];
          for(i=1;i<numTopics;i++){
            window['numSubtopics'+i] = document.getElementById('subheadingScroll'+i).childElementCount;
            loop = loop.concat(window['numSubtopics'+i]);
          }
/*
        var numSubtopics1 = 4;
        var numSubtopics2 = 5;
        var numSubtopics3 = 2;
        var numSubtopics4 = 3;
        var numSubtopics5 = 4;
        
        var loop = [numSubtopics1,numSubtopics2,numSubtopics3,numSubtopics4,numSubtopics5];
*/
  //creates variable to get Elements based on number in 'numTopics'    var named sec#
       var secList = [];               // [sec1,sec2,sec3,sec4 ...]
       for (i=1;i<=numTopics;i++){
          window["sec"+i] = document.getElementById('sec'+i);     //create get elements (sec#)
          secList[i-1] = window["sec"+i];
        }
        
  //defines width of top bar sections depending on # of sections
      var numSecs = document.getElementById('sec1').parentElement.childElementCount;
      var numSecCalc = 100/numSecs;
      for(i=0;i<secList.length;i++){
        document.getElementById("sec"+(i+1)).style.width = numSecCalc+'%';
      }
  
  
////click subheading text on 2nd bar to scroll to it's section (fully modular)////

    var scr = window.scrollY;  //accounts for current scroll position to calc top or subSection headings
    var bT = (document.getElementById('scrollBar').scrollHeight*2);  //height of both top bar and subheading bar
        
  //fill array subList with strings to get Element names in for loop
    var subList = [];
    var getSubList = [];
    for (i=1;i<numTopics;i++){              //for # of topics listed in numTopics, loop:
          var numSubtopic = loop[i-1];
          getSubList[i-1] = new Array;
          for(k=1;k<=numSubtopic;k++){
            subList = subList.concat("sub"+k+i);
            window["sub"+k +i] = document.getElementById("sub"+k +i);
            getSubList[i-1].push(window["sub"+k +i]);
          }
        }
   
    //defines var for locating top of each subsection        var named topSec##  and filling array with those variables
        var topSecList = [];
        for (i=1;i<numTopics;i++){              //for # of topics listed in numTopics, loop:
          var numSubtopic = loop[i-1];
          for(k=1;k<=numSubtopic;k++){
            window["topSec"+k +i] = document.getElementById("subDes"+k +i).getBoundingClientRect().top + scr -bT;
            topSecList = topSecList.concat(window['topSec'+k+i]);
          }
        }
        
        for(i=0;i<subList.length;i++){
          (function(i){                     //scoping function AHHHHH!!
            var elem = document.getElementById(subList[i]);
            elem.addEventListener("click",function(){window.scrollTo(0,(topSecList[i])); },false );
          })(i)
        }



          
////click to scroll to different main topics (fully modular)////

        var topBar = document.getElementById('scrollBar').offsetTop;  //defines starting scroll position of top menu
        var description = document.getElementById('description');
        
    //declares 'relativeBox' variables for 'click to scroll to'.
    var relativeOffset = [];
    var relativeScroll = [];
        for (i=1;i<=loop.length;i++){
          window['relativeBox'+i+'Offset'] = document.getElementById('relativeBox'+i).offsetTop;   // var relativeBox1Offset
          relativeOffset = relativeOffset.concat(window['relativeBox'+i+'Offset']);
          window['relativeBox'+i+'Scroll'] = document.getElementById('relativeBox'+i).scrollHeight;   //var relativeBox1Scroll
          relativeScroll = relativeScroll.concat(window['relativeBox'+i+'Scroll']);
        }
  
          sec1.onclick = function(){    //standard
            window.scrollTo(0,topBar);
          }
          sec2.onclick = function(){    //must have at least 1 main topic
            window.scrollTo(0,description.offsetTop + description.scrollHeight);
          }
          
          for (i=2;i<secList.length;i++){
            (function(i){
            var elem = secList[i];
            elem.addEventListener('click', function(){
              //alert(i);
              window.scrollTo(0,(relativeOffset[i-2] + relativeScroll[i-2]));
              },false);
            })(i)
          }

          
          
////Fully modular change opacity function. bolds first parameter and makes all others more opaque////
        function changeOpacity(){
          remArray = [];
          add1 =arguments[0];
          for(i=1;i<arguments.length;i++){
            window['rem'+i] = arguments[i]; //if avoid global variables, will have to manually create array = []; and add rem1,rem2,rem3 ... via for loop
            remArray = remArray.concat(arguments[i]);
          }
            add1.classList.add('highOpacity'); //add bold to 'overview'
            add1.classList.remove('lowOpacity');
            for(i=0;i<remArray.length;i++){   //remove bold from all others
              remArray[i].classList.remove('highOpacity');
              remArray[i].classList.add('lowOpacity');
            }
          }
          
          
        window.onscroll = function() {    //should move all variables outside onscroll function
          
            (function(){
          
      //scroll to fix for top bar          standard
          var pageTop = document.body.scrollTop;  //defines the scroll position at top of window
          var scrollBar = document.getElementById('scrollBar');
          if (pageTop > topBar){
            scrollBar.classList.add('addFix');
            scrollBar.classList.remove('goBack');
          }
          else {
            scrollBar.classList.remove('addFix');
            scrollBar.classList.add('goBack');
          }
          
      //standard variables
          var pageTopPlus = document.body.scrollTop + scrollBar.scrollHeight;
          var pageTopPlusPlus = document.body.scrollTop + (scrollBar.scrollHeight*2.6); //2.6, idk, just worked that way. may fail later :/
          
  
         //creates variable to get Elements based on number in 'numTopics'    var named subheadingScroll#
          var subheadingScroll = [];
       for (i=1;i<=numTopics;i++){
          window["subheadingScroll"+i] = document.getElementById('subheadingScroll'+i);     //create get elements (subheadingScroll#)
          subheadingScroll = subheadingScroll.concat(window['subheadingScroll'+i]);
        }
        
  ///////// sets the width of sub-bar section headings depending on the number of sections
      for(i=0;i<subheadingScroll.length-1;i++){
        var numSec2 = subheadingScroll[i].childElementCount;
        var numSecCalc2 = 100/numSec2;
        var children = subheadingScroll[i].children;
        for(k=0;k<numSec2;k++){
          var current = children[k];
          current.style.width = numSecCalc2+'%';
        }
      }

///// changes scroll/fix positioning of subheading bars at certain scroll heights (Fully Modular) ////


          for(i=0;i<numTopics-1;i++){
            
            topBar2 = relativeOffset[i];                        //topFirstBar
            topBarPlus = topBar2 + relativeScroll[i];       //topBarPlus
            thisBar = subheadingScroll[i];                     //firstBar
            
          if (pageTopPlus < topBar2){                                //if top of page above subheading bar.
            thisBar.classList.add('goBack1');
            thisBar.classList.remove('goBack12');
            thisBar.classList.remove('addSubFix');
          }
          else if (pageTopPlus > topBar2 && pageTopPlusPlus < topBarPlus){    //if top of page within subsection area.
            thisBar.classList.add('addSubFix');
            thisBar.classList.remove('goBack1');
            thisBar.classList.remove('goBack12');
          }
          else if (pageTopPlusPlus > topBarPlus){                   //if top of page below subheading bar.
            thisBar.classList.add('goBack12');
            thisBar.classList.remove('goBack1');
            thisBar.classList.remove('addSubFix');
          }
            
          }
          
          
      //change opacity of top bar headings based on scroll position
          var topic1Dist = (description.offsetTop + description.scrollHeight -1);    //standard (must have at least 1 subheading)
          
          var topicDistList = [topic1Dist];
          
      //creates variable # of 'topic#Dist' vars.
          for(i=0;i<relativeOffset.length-1;i++){
            window['topic'+(i+2)+'Dist'] = (relativeOffset[i] + relativeScroll[i] -1);
            topicDistList[i+1] = (window['topic'+(i+2)+'Dist']);
          }
        
          
        var topicDistList2 = []; // new Array containing conditional statements and page heights
  //fills first and last positions (standard)
          topicDistList2[0] = pageTop < topic1Dist;   //topic1Dist set to beginning of array
          topicDistList2[numTopics-1] = pageTop > topicDistList[topicDistList.length-1];    //last element in topicDistList array set to last element in array
  //fills middle positions
          for (i=1;i<numTopics-1;i++){
            topicDistList2[i] = pageTop > topicDistList[i-1] && pageTop < topicDistList[i];  //fill rest of elements in topicDistList2 array
          }
          
  //create array with different arrangements of sec#.
        var secListArr = [];
        for(i=0;i<secList.length;i++){
          secListArr[i] = secList.slice(0);
          roo2 = secListArr[i].splice(i,1)[0];
          secListArr[i].splice(0,0,roo2);
        }
          
  //loop to change opacity of top bar
          for (i=0;i<topicDistList2.length;i++){
            if (topicDistList2[i]){
              changeOpacity.apply(this,secListArr[i]);
            }
          }


/* change opacity of subBar on scroll position */

  //create array with different arrangements of sec#.

        var subListArr = [];
        for(i=0;i<getSubList.length;i++){
          subListArr[i] = [];
          for (k=0;k<getSubList[i].length;k++){
            subListArr[i][k] = getSubList[i].slice(0);
            roo2 = subListArr[i][k].splice(k,1)[0];
            subListArr[i][k].splice(0,0,roo2);
            }
        }

//alert(getSubList[0]); -> [sub11,sub21,sub31,sub41]; 4 arrangements of this
//alert(getSubList[1]); -> [sub12,sub22,sub32,sub42]; 4 arrangements of this
//alert(getSubList[2]); -> [sub13,sub23,sub33,sub43]; 4 arrangements of this

        var x = 1;
        var topicSecParse = [];
        for(i=0;i<numTopics-1;i++){   //number of non-overview topics   0,1,2
          subTop = loop[i]-1;
          topicSecParse[i] = [];
          topicSecParse[i][0] = pageTop < topSecList[x]-1;            //defines first element of every 'topic array'
          topicSecParse[i][subTop] = pageTop > topSecList[x+loop[i]-2]-1;   //defines last element of every 'topic array'  wont work if have 1 subheading
          x+= loop[i];
        }

        var z = 1;
        for(i=0;i<numTopics-1;i++){
          for(k=1;k<loop[i]-1;k++){
            topicSecParse[i][k] = pageTop > topSecList[z]-1 && pageTop < topSecList[z+1]-1;
            z += 1;
          }
          z -= (loop[i]-2);
          z += loop[i];
        }
        
 
          
          for (j=0;j<numTopics-1;j++){
            for (n=0;n<loop[j];n++){
              if (topicSecParse[j][n]){
                changeOpacity.apply(this,subListArr[j][n]);
              }
            }
          }
 
        })();   //end of window scroll function
        }
        
})();
      }
           //end of script
      
    </script>
    
    <style>
    *{box-sizing:border-box;
    }


    .addFix{       /*add fix position for main bar*/
      position:fixed;
      top:0;
    }
    .addSubFix{     /*add fix position for subheading bars*/
      position:fixed;
      top:7vh;
    }
    .goBack{      /*top position for main bar*/
      position:absolute;
      top:20vh;
    }

    .goBack1{    /*top position for subheading bars*/
      position:absolute;
      top:0;
    }
    .goBack12{   /*bottom position for subheading bars*/
      position:absolute;
      bottom:0;
    }
    
    
    #subject{
      width:98vw;
      height:15vh;
      text-align:center;
      font-size:3vmin;
      //border:1px solid black;
      //font-family:garamond;
    }
    #subject h1{
      padding-top: 2.5%;
    }

    .scrollFixBar{
      z-index:10;
      background-color:white;
      width:98vw;
      float:left;
      margin-left:0;
      height:7vh;
      border-top:1px solid black;
      border-bottom:1px solid black;
      margin-bottom:5%;
      text-align:center;
    }
    .topics{                                     /*defines font size and top bar div width*/
      position:relative;
      height:100%;
      display:inline-block;
      font-size: 1.3vmin;
      cursor:pointer;
      border-right:0.8px solid black;
    }
    .topics h2{
      position:absolute;
      top:50%;
      left: 50%;
      transform: translateY(-50%) translateX(-50%);
      margin:auto;
    }
    #description{
      margin-top:15vh;
      width:98vw;
      float:left;
      position:relative;
      font-size:1.3vmax;
    }
    .topicOverview{
      float:left;
      width:90%;
      margin-left:3%;
      margin-bottom:10%;
      line-height:80%;
      position:relative;
    }
    .topicOverview p {
      font-size:2vmax;
      line-height:110%;
    }
    .topicOverview h2 {
     text-align:center;
     font-size:2vmax;
    }
      .relativeBoxes{
      position:relative;
      float:left;
      width:100%;
      height:100%;
    }
    .topicHeader{
      width:95%;
      float:left;
      text-align:center;
      border-bottom:1px solid black;
      margin:5% 0 5% 3%;
      position:relative;
      font-size:2vmax;
    }
    .topicDescription{
      position:relative;
      border:1px solid black;
      text-align:center;
      width:60vw;
      float:left;
      margin-left:19vw;
      padding:1em;
      margin-bottom:5%;
      font-size:1.3vmax;
    }
    .topicDescription p{
      font-size:1.5em;
    }
    .lowOpacity{
      opacity:0.4;
    }
    .highOpacity{
      opacity:1;
    }
    #withPic{
      display:inline-block;
    }
    #withPic p {
      width:42vw;
      float:left;
    }
    #withPic img {
      width:42vw;
      float:right;
    }
    
    </style>
  </head>

  <body>
    
    <div id='subject'>
      <h1>Exam 4</h1>
    </div>
    
    <div id='scrollBar' class='scrollFixBar goBack' style='z-index:100'>
      <div id='sec1' class='topics highOpacity'>
        <h2>Overview</h2>
      </div
      ><div id='sec2' class='topics highOpacity'>
        <h2>RNA Motility & Decay</h2>
      </div
      ><div id='sec3' class='topics lowOpacity'>
        <h2>Ribosome & Initiation</h2>
      </div
      ><div id='sec4' class='topics lowOpacity'>
        <h2>Translation Process</h2>
      </div
      ><div id='sec5' class='topics lowOpacity'>
        <h2>Ribosome Biogenesis</h2>
      </div
      ><div id='sec6' class='topics lowOpacity'>
        <h2>Proteomics</h2>
      </div
      ><div id='sec7' class='topics lowOpacity'>
        <h2>Regulatory Small RNAs</h2>
      </div
      ><div id='sec8' class='topics lowOpacity'>
        <h2>lincRNAs</h2>
      </div>
    </div>
    
  <div id='description'> <!--auto-add topic descriptions? -->
    <div class='topicOverview'>
      <h2>RNA Motility and Decay</h2>
      <p>* All RNAs that are exported to the cytoplasm go through the <b>nuclear pore complex</b>, but different types of RNAs use different pathways. A general export receptor interacts with the RNA either directly or through <b>adapter proteins</b>.</p>
      <p>* The <b>directionality</b> for exportin-mediated export is provided by cycles of GTP binding and hydrolysis by the <b>Ran protein</b>. The mRNA export pathway may use cycles of phosphorylation of an adapter <b>SR protein</b> to achieve directionality. A DEAD-box ATPase may also give directionality.</p>
      <p>* It is becoming apparent that many RNAs are further <b>localized within the cytoplasm</b>. This may have profound effects on the coordination and regulation of gene expression.</p>
      <p>* <b>RNA decay is critical for gene regulation</b> and plays other roles. The basic process of RNA decay is: deadenylation followed either by 1) continued exonuclease digestion 3’ to 5’ by the exosome, or 2) decapping. If decapped, the mRNA is typically degraded 5’ to 3’ by Xrn1.</p>
    </div>
    <div class='topicOverview'>
      <h2>Ribosome & Initiation</h2>
      <p>* The ribosome consists of two subunits, the large and small subunits, which are composed of RNA and protein. The large subunit includes the peptidyl transferase center and the small subunit decodes mRNA.</p>
      <p>* Prokaryotic and eukaryotic translation initiation have some common features. A charged initiator tRNA forms a complex with the small subunit that recognizes a start codon, followed by joining of the large subunit, GTP hydrolysis, and ultimately initiation.</p>
      <p>* Eukaryotic initiation has unique features; most is <b>cap-dependent</b> and occurs by <b>scanning</b>, the <b>mRNA circularizes</b>, and there are more factors. Some mRNAs (especially viral) use <b>IRES</b> elements.</p>
      <p>* Prokaryotes and eukaryotes have varied mechanisms for <b>control of translation</b>. These mechanisms commonly use <b>RNA-RNA interactions</b> and RNA structures.</p>
    </div>
    <div class='topicOverview'>
      <h2>Translation Process</h2>
      <p>* All organisms use essentially the same genetic code; three nucleotides comprise a codon, which specifies an amino acid or signals termination.</p>
      <p>* Translation elongation consists of cycles of 1) multi-step binding of aminoacyl tRNA in a ternary complex with <b>EF-Tu and GTP</b>; 2) peptidyl transfer, catalyzed by the 50S subunit; and 3) translocation, which is greatly facilitated by <b>EF-G and GTP</b>.</p>
      <p>* Termination is mediated by the appropriate stop codons and <b>release factors</b>, which recognize them. The release factors bind in the <b>A-site</b> and may contribute directly to hydrolysis to liberate the peptide chain.</p>
      <p>* Complex systems are present to ensure the degradation of damaged mRNAs that are unable to be translated accurately into protein. These pathways include <b>nonsense-mediated decay</b> (NMD), <b>nonstop decay</b>, and <b>no-go decay</b>.</p>
    </div>
     <div class='topicOverview'>
      <h2>Regulatory Small RNAs</h2>
      <p>* Small RNAs, in the forms of <b>siRNAs</b> and <b>miRNAs</b>, play large roles in the regulation of <b>gene expression</b> in eukaryotes. They are important in normal cell metabolism, development, and defense against invaders.</p>
      <p>* siRNAs are produced from longer segments of dsRNA by <b>Dicer</b>, assembled into <b>RISC</b>, and targeted to mRNAs with perfect complementarity, giving silencing by cleavage and degradation of the RNA or by formation of heterochromatin</p>
      <p>* The pathway for miRNA has many steps in common with that for siRNA. However, miRNAs are processed from <b>hairpin precursors</b>, and they most often have imperfect complementarity with their targets, giving effects on <b>translation</b> and on <b>mRNA decay</b> by standard pathways rather than <b>Ago-mediated cleavage</b> of the mRNA</p>
    </div>
    <div class='topicOverview'>
      <h2>lincRNAs</h2>
      <p>* Chromosomes are arranged to allow <b>long-range contacts</b> between regulatory elements and promoters, gene ends, and much longer-range contacts involving multiple genes. Contacts maps have led to the view that chromosomes are arranged in a segmented way.</p>
      <p>* Genomic methods have indicated the presence of large numbers of <b>long, intervening non-coding RNAs</b> (lincRNAs). While many of these RNAs are likely to be non-functional, some are known to be functional and many others are likely to be functional as well.</p>
      <p>* Some lncRNAs act <b>in cis</b> on DNA adjacent to their transcription sites, while others function <b>in trans</b> to regulate gene expression, functioning in the nucleus or the cytoplasm.</p>
      <p>* More functions almost certainly exist and are yet to be discovered.</p>
    </div>
  </div>
    
    <div class='topicHeader'>
      <h1>RNA Motility and Decay</h1>
    </div>
    <div class='topicDescription'>
      <p>* All RNAs that are exported to the cytoplasm go through the nuclear pore complex, but different types of RNAs use different pathways. A general export receptor interacts with the RNA either directly or through adapter proteins.</p>
      <p>* The directionality for exportin-mediated export is provided by cycles of GTP binding and hydrolysis by the Ran protein. The mRNA export pathway may use cycles of phosphorylation of an adapter SR protein to achieve directionality. A DEAD-box ATPase may also give directionality.</p>
      <p>* It is becoming apparent that many RNAs are further localized within the cytoplasm. This may have profound effects on the coordination and regulation of gene expression.</p>
      <p>* RNA decay is critical for gene regulation and plays other roles. The basic process of RNA decay is: deadenylation followed either by 1) continued exonuclease digestion 3’ to 5’ by the exosome, or 2) decapping. If decapped, the mRNA is typically degraded 5’ to 3’ by Xrn1.</p>
    </div>
    
  <div id='relativeBox1' class='relativeBoxes'>
    <div id='subheadingScroll1' class='scrollFixBar goBack1'>
      <div id='sub11' class='topics'>
        <h2>RNA Export</h2>
      </div
      ><div id='sub21' class='topics'>
        <h2>RNA Localization</h2>
      </div
      ><div id='sub31' class='topics'>
        <h2>Constituitive RNA Decay</h2>
      </div>
    </div>
    
    <div class='topicOverview' style='margin-top:15vh' id='subDes11'>
          <h2>RNA Export</h2>
      <p><i><u>Nuclear Pore Complex</u></i></p>
      <p>NPC is a large pore with fibrils pointing towards the cytoplasm and a basket facing the nucleus. Composed of 30 different pore proteins. 8-30 copies per complex.</p>
      <p>These NPCs are highly efficient <b>Sorting Machines</b></p>
      <p></p>

      <p><i><u>Export of tRNA</u></i></p>
      <p>tRNA binds directly to a <b>karyopherin</b>, which recognize Nuclear Export Signals (NES) and Nuclear Localization Signals (NLS). Most are peptide sequences, but can be RNA</p>
      <p>After being processed tRNA binds exportin-t. Binding depends on tRNA structure, so this may be a quality control step</p>
      <p>Karyopherins interact with Ran, a GTPase that cycles through GTP binding and hydrolysis to transport things into and out of the nucleus.</p>
      <p>Binding of Ran-GTP to exportin-t favors the closed state, which binds tRNA tightly and specifically</p>
      <p>GTPase activating protein is cytoplasmic (RanGTP), as well as helper RanBP1/2 proteins.</p>
      <p>GDP-GTP exchange factor is nuclear (RanGEF, or RCC1). Therefore, Ran is mostly GTP-bound in the nucleus and mostly GDP-bound in the cytoplasm</p>
      <p>Exportin-t binds Ran-GTP, so nuclear binding and export are favored</p>
  
      
      <p><i><u>Adaptor Proteins</u></i></p>
      <p>Export of microRNAs are analogous to tRNAs, but the receptor is Exportin-5 instead of Exportin-t</p>
      <p>Export receptor for snRNAs is CRM1 (or exportin-1), which requires a cap-binding complex and NES-containing adaptor protein.</p>
      <p>After export, snRNAs are processed, assembled with Sm proteins, and re-imported.</p>
      
      <p><b>TAP</b> is the transport factor for metazoan mRNAs</p>
      <p>TAP binds to <b>ALY/REF adaptor protein</b>, which is a component of transcription and splicing complexes.</p>
      <p>Adaptor proteins are diverse (help export snRNAs, miRNAs, tRNAs, and mRNAs)</p>
      
      <p><i><u>SR Proteins</u></i></p>
      <p>SR Proteins can also function as adaptors (and replace ALY/REF in some cases)</p>
      <p>SR protein phosphorylation may confer directionality to export</p>
      <p>RNP remodeling by DEAD-box protein may confer directionality to export. DEAD-box protein localizes only to cytoplasmic face</p>
    </div>
    
    <div class='topicOverview' id='subDes21'>
          <h2>RNA Localization</h2>
      <p><i><u>mRNA localization important in development</u></i></p>
      <p>Many RNAs are localized in Fly Embryos</p>
      <p>71% of over 3000 genes display a localization pattern</p>
      
      <p><i><u>Motor-mediated mRNA Transport</u></i></p>
      <p>Transport can be mediated by kinesin, processive myosin, or dynein motors</p>
      <p>Localization mainly depends on: RNA sequences termed "zipcodes", and RNA-binding proteins</p>
      <p>Non-motor mediated transport includes: anchoring and selective degradation</p>
      
      <p><i><u>Advantages of mRNA localization</u></i></p>
      <p>Provides a mechnism for gene expression to be spacially restricted</p>
      <p>Local stimulii can regulate translation "on-site", without sending signal to nucleus or throughout the cell</p>
      <p>More efficient to transport single mRNA, which can then synthesize many molecules of localized protein</p>

    </div>
    
    <div class='topicOverview' id='subDes31'>
          <h2>Constituitive RNA Decay</h2>
      <p><i><u>Overview</u></i></p>
      <p>mRNA degradation is important for:<br>
      Functions in gene expression by rapidly changing the amount of an mRNAavailable for translation
      <br>Functions in quality control, principally associated with translation to ensure that mRNA pool is translatable
      <br>Functions in defense against viruses</p>
      <p>mRNA degradation process:
      <br>1) Deadenylation
      <br>2) 3-prime to 5-prime Exonuclease
      <br>3) 5-prime Decapping
      <br>5-prime to 3-prime exonuclease (Xrn1p)</p>
      
      <p><i><u>Deadenylation</u></i></p>
      <p>Deadenylation removes poly-A tail. There are 3 major pathways</p>
      <p>(1)Main deadenylation complex in yeast contains two nucleases (Ccr4p and Pop2p).Complex is inhibited by Poly-A binding protein. Ccr4p is highly conserved among eukaryotes</p>
      <p>Ccr4p possesses deadenylation activity and is central to the activity of the complex. Isolation of Ccr4p co-purifies with deadenylation activity, which is not dependent on Pop2</p>
      <p>(1)<b>PAN is a second deadenylation complex</b>. Its activity becomes significant when Ccr4p is deleted. And it is stimulated by Poly-A binding protein</p>
      <p>Initial Poly-A shortening my be coupled to nuclear export</p>
      <p>(3) PARN my be the major deadenylase in Mammals (absent in yeast and flies)</p>
      <p>PARN complexes may be differentially regulated.
      <br>Flux through different mRNP states could alter susceptibility to different deadenylation complexes, allowing control</p>
      
      <p><i><u>3-prime to  5-prime Decay</u></i></p>
      <p>3 to 5-prime exonuclease forms a hexameric ring with three additional subunits that form a cap.
      <br>Rrp44(the 10th subunit) is responsible for most of the nucleolytic activity
      <br>Activity of exosome depends on RNA helicase protein (Ski2) and ATP</p>
      <p>Thought that substrate is fed through the ring channel to Rrp44 for degradation</p>
      
      <p><i><u>Decapping</u></i></p>
      <p>If mRNA is not decayed by previous exosome, it is fed through an alternative decap pathway that starts with decapping</p>
      <p>Decapping proteins (Dcp1/2) are highly conserved. Dcp2 is the catalytic subunit
      <br>Decapping is achieved by hydrolysis, yielding GDP and 5-prime phosphorylated mRNA, a good substrate for 5-to-3-prime exonuclease</p>
      <p><b>Poly-A tail inhibits Decapping</b> through binding to Pab1p. Either forms a ring (cap and tail) through translation or another binding protein and inhibits decapping enzyme from reaching the cap.</p>
      <p>P bodies: processing bodies for RNA within the cytoplasm</p>
      
      <p><i><u>5-prime to 3-prime Decay</u></i></p>
      <p>After decapping (alternative route) final step is 5-to-3-prime decay. This is mediated by the <b>Xrn1 protein</b>, which binds RNA<br>
      RNA end is buried deep within the active site, thus "trapping" the RNA
      <br>The enzyme recongnizes the 5-prime phosphate and removes one nucleotide at a time.</p>
    </div>
    
  </div>  <!--relativeBox1-->
    
    <div class='topicHeader'>
      <h1>Ribosome Structure & Initiation</h1>
    </div>
    <div class='topicDescription'>
      <p>* The ribosome consists of two subunits, the large and small subunits, which are composed of RNA and protein. The large subunit includes the peptidyl transferase center and the small subunit decodes mRNA.</p>
      <p>* Prokaryotic and eukaryotic translation initiation have some common features. A charged initiator tRNA forms a complex with the small subunit that recognizes a start codon, followed by joining of the large subunit, GTP hydrolysis, and ultimately initiation.</p>
      <p>* Eukaryotic initiation has unique features; most is cap-dependent and occurs by scanning, the mRNA circularizes, and there are more factors. Some mRNAs (especially viral) use IRES elements.</p>
      <p>* Prokaryotes and eukaryotes have varied mechanisms for control of translation. These mechanisms commonly use RNA-RNA interactions and RNA structures.</p>
    </div>
    
  <div id='relativeBox2' class='relativeBoxes'>
    <div id='subheadingScroll2' class='scrollFixBar goBack1'>
      <div id='sub12' class='topics'>
        <h2>Molecules of Translation</h2>
      </div
      ><div id='sub22' class='topics'>
        <h2>Prokaryotic Initiation</h2>
      </div
      ><div id='sub32' class='topics'>
        <h2>Eukaryotic Initiation</h2>
      </div
      ><div id='sub42' class='topics'>
        <h2>Prokaryotic Initiation Control</h2>
      </div
      ><div id='sub52' class='topics'>
        <h2>Eukaryotic Initiation Control</h2>
      </div>

    </div>
    
    <div class='topicOverview' style='margin-top:15vh' id='subDes12'>
          <h2>Molecules of Translation</h2>
      <p><i><u>Early Ribosome detection</u></i></p>
      <p>First imaged by EM in 50s</p>
      <p>Proteins localized by binding antibodies</p>
      <p>Most structural work done on prokaryotic ribosomes</p>
      
      <p><i><u>Sequence Analysis to predict Secondary Structures</u></i></p>
      <p>About 60% of rRNA nucleotides are base paired</p>
      <p>Sequence analysis used to predict the secondary structure of rRNA by finding base-pairing potential
      <br> When multiple related sequences are available, <b>covariation</b> provides additional evidence for pairing</p>
      <p>97% of predicted base pairs were present in crystal structure<br>75% of base pairs in crystal structures were predicted, others not detectable</p>
      <p>Some tertiary interactions show covariation and can therefore be predicted
      <br>Many tertiary contacts are mediated by Adenosine nucleotides</p>
      
      <p><i><u>30S and 50S</u></i></p>
      <p>30S secondary structure has a central domain that remains intact
      <br>Overall tertiary arrangement is dominated by rRNA</p>
      <p>50S structure is monolithic, not modular
      <br>Most of protein mass is on or near surface
      <br>Portions of proteins toward middle are in unprecedented, unfolded conformations; threaded through RNA</p>
      
      <p><i><u>30S 50S Interaction</u></i></p>
      <p>Most of intersubunit bridges are RNA
      <br>All bridges near tRNA are RNA
      <br>Translocation requires large movements, some bridges may be dynamic</p>
      <p><b>Ribosome Model is based on Structural Information</b>
      <br>There is a large cavity between subunits to accomodate the three tRNAs
      <br>tRNAs interact with 30S subunit through anticodon ends and bind to mRNA, also bound to 30S
      <br>tRNAs interact with 50S through acceptor stems. This is where peptidyl transfer happens</p>
      <p><b>Codon-anticodon base pairing</b>
      <br>Bend in mRNA between A and P sites allows adjacent tRNAs to bind to consecutive codons in proper orientations for peptidyl transfer</p>
      <p><b>50S structure shows no proteins near active site for pepidyl transfer</b>
      <br>The ribosome is a ribozyme, RNA is catalytic here.
      <br>Proteins snake towards, but not into, the active site</p>
      
      <p><i><u>Charging tRNAs</u></i></p>
      <p>Aminoacyl-tRNA synthetase enzymes carry out a two-step reaction
      <br><b>First</b>, the amino acid is activated by reacting with ATP to give aminoacyl-AMP
      <br><b>Second</b>, the amino acid is transferred to the 3-prime end of the correct tRNA, with displacement of AMP</p>
      <p><b>Extensive interactions between tRNA and its Aminoacyl tRNA Synthetase</b>
      <br>Recognition occurs between the enzymes and various parts of the tRNA:
      <br>Acceptor Stem (upstream from common region)
      <br>Anticodon loop
      <br>D-stem</p>
      <p>Structure showed extensive contacts between enzyme and tRNA in all regions known to function in specificity
      <br>Aminoacyl-tRNA Synthetases also must recognize the correct amino acid and exclude incorrect amino acids</p>
    </div>
    
    <div class='topicOverview' id='subDes22'>
          <h2>Prokaryotic Initiation</h2>
      <p><i><u>Prokaryotic gene structure</u></i></p>
      <p><b>Prokaryotes have multiple ORFs in one transcript whereas eukaryotes do not</b>
      <br>Prokaryotic transcripts are polycistronic, eukaryotic are monocistronic</p>
      <p>Translation starts just <b>Downstream from Shine-Dalgarno Sequence</b>
      <br>16S rRNA pairs with Shine-Dalgarno Sequence</p>
      <p><b>The Initiator tRNA is charged with N-formyl-methionine</b>
      <br> Analysis of linkage between amino acid and tRNA led to discovery of tRNA-fMet</p>
      
      <p><i><u>Steps of Translation Initiation in Prokaryotes</u></i></p>
      <p><b>Experiment:</b> Mixed P32-IF2 with H3-fMet-tRNA and AUG with 30S
      <br>Non-hydolyzable analog GDPCP supported IF2 Binding
      <br>Assayed by sucrose gradient ultracentrifugation</p>
      <p>Neither IF2 not 70S ribosomes alone give efficient GTP hydrolysis
      <br>Only in combination is strong hydrolysis observed
      <br>GTP hydrolysis leads to removal of IF2</p>
      <p>GTP hydrolysis is required for release of IF2
      <br>Much more fMet-tRNA bound and formed 70S initiation complex in presence of GTP, as analyzed by sucrose gradient</p>
    </div>
    
    <div class='topicOverview' id='subDes32'>
          <h2>Eukaryotic Initiation</h2>
      <p><i><u>Cap-dependent Initiation</u></i></p>
      <p><b>Most eukaryotic translation is cap dependent and occurs by scanning mechanism</b>
      <br>Eukaryotic initiation is with Met, not fMet, but still uses a different tRNA
      <br>Translation usually begins from the 5-prime most AUG sequence</p>
      <p>Scanning model developed based on these findings:
      <br>Translation was not known to begin from internal AUG
      <br>Initiation did not occur at a fixed distance from the 5-prime end
      <br>Translation is greately facilitated by the presence of the cap</p>
      <p><b>Secondary structure in 5-prime UTR has effect on translation initiation</b>
      <br>Hairpins formed around AUG site can inhibit initiation</p>
      
      <p><i><u>Steps of eukaryotic translation initiation</u></i></p>
      <p>look at diagram</p>
      <p><b>Components of Cap-binding complex</b>
      <br>eIF4F complex now defined as including tow proteins:
      <br>eIF4E, which binds cap directly, and
      <br>eIF4G, a large protein that binds several other proteins</p>
      
    <p>eIF4A, a DEAD-box protein, can also associate, and has RNA unwinding activity and is stimulated by eIF4B</p>
      
      <p><i><u>Internal Ribosome Entry Site (IRES)</u></i></p>
      <p>Found in some viral RNAs</p>
      <p>EMCV virus particle recruited through direct interaction of IRES and eIF4G
      <br>HCV virus has an IRES RNA that specifically recognizes 40S and eIF3
      <br>CrPV virus associates directly with 40S; portion of IRES mimics tRNA in P site</p>

    </div>
    
    <div class='topicOverview' id='subDes42'>
          <h2>Prokaryotic Initiation Control</h2>
      <p><i><u>Small RNAs Regulate Translation in Prokaryotes</u></i></p>
      <p>Mre than 40 sRNAs identified in E.coli
      <br>only a handful have been characterized, but these bind by base-pairing to mRNA targets and regulate translation</p>
      <p><b>Example:</b> Can bind to break secondary structure of mRNA forming a hairpin with itself, thus freeing its shine dalgarno sequence and increasing translation
      <br>Others can repress translation</p>
      
      <p><i><u>Riboswitches: A common prokaryoteic mechanism of translation regulation</u></i></p>
      <p>Example: Bindingof TPP riboswitch represses the translation of mRNAs that encode enzymes involved in TPP synthesis
      <br>Biochemical studies performed to show clearly that TPP binds to RNA element, including conformational change that blocks Shine-Dalgarno sequence</p>
      <p><b>Mechsnism of regulation by riboswitches</b>
      <br>Binding of ligand induces structure that sequesters the Shine-algarno sequence
      <br>Riboswitches are ubiquitous in prokaryotes, ligands include
      <br>Vitamins
      <br>Amino acids
      <br>Nucleobases
      <br>Sugars
      <br>Metal ions</p>
      <p>Many function by regulating transcription</p>
      
    </div>
    
    <div class='topicOverview' id='subDes52'>
          <h2>Eukaryotic Initiation Control</h2>
      <p><i><u>Phosphorylation of Initation factor eIF2alpha</u></i></p>
      <p>Used extensively in reticulocytes, which basically make only hemoglobin
      <br>If cell is starved for iron, heme-controlled repressor phosphorylates one of the subunits of eIF2alpha
      <br>Phosphorylated eIF2alpha binds eIF2B tightly, which sequesters eIF2B such that it is unable to promote GDP/GTP exchange</p>
      <p>This leads to global repression of translation</p>
      
      <p><i><u>Translation repression by protein binding to 3-prime UTR</u></i></p>
      <p>Several examples in development where 3-prime UTR binding of protein X, in some with intermediate protein Y, inhibits translation</p>
      <p>Inhibition involves interaction with Cap</p>
      
      <p><i><u>miRNA: a major pathway for translational regulation in eukaryotes</u></i></p>
      <p>Binds to mRNA to block translation or cleave the transcipt.
      <br>We will talk more about this later</p>
  
    </div>
    
  </div> <!--relative box-->
    
        <div class='topicHeader'>
      <h1>Translation Process</h1>
    </div>
    <div class='topicDescription'>
      <p>* All organisms use essentially the same genetic code; three nucleotides comprise a codon, which specifies an amino acid or signals termination.</p>
      <p>* Translation elongation consists of cycles of 1) multi-step binding of aminoacyl tRNA in a ternary complex with EF-Tu and GTP; 2) peptidyl transfer, catalyzed by the 50S subunit; and 3) translocation, which is greatly facilitated by EF-G and GTP.</p>
      <p>* Termination is mediated by the appropriate stop codons and release factors, which recognize them. The release factors bind in the A-site and may contribute directly to hydrolysis to liberate the peptide chain.</p>
      <p>* Complex systems are present to ensure the degradation of damaged mRNAs that are unable to be translated accurately into protein. These pathways include nonsense-mediated decay (NMD), nonstop decay, and no-go decay.</p>
    </div>
    
  <div id='relativeBox3' class='relativeBoxes'>
    <div id='subheadingScroll3' class='scrollFixBar goBack1'>
      <div id='sub13' class='topics'>
        <h2>Determining the Genetic Code</h2>
      </div
      ><div id='sub23' class='topics'>
        <h2>Elongation</h2>
      </div
      ><div id='sub33' class='topics'>
        <h2>Termination</h2>
      </div
      ><div id='sub43' class='topics'>
        <h2>Quality Control</h2>
      </div>

    </div>
    
    <div class='topicOverview' style='margin-top:15vh' id='subDes13'>
          <h2>Determining the Genetic Code</h2>
      <p><i><u>Proof of a Triplet Code</u></i></p>
      <p><b>Experiment:</b> Used synthetic RNAs with repeating sequences, were able to produce polypeptides
      <br>repeating dinucleotide produced repeating dipeptide
      <br>repeating trinucleotide produced mixture of three homopeptides
      <br>repeating tetranucleotides produced a tetrapeptide</p>

      
      <p><i><u>A tRNA Binding Assay to help break the code</u></i></p>
      <p><b>Experiment:</b>
      <br>Different Aminoacylated tRNAs were bound to ribosomes in the presence of different tri-nucleotides</p>
      <p>Made a bunch of nucleotide repeats to create codon map</p>
      
      <p><i><u>Base-pairing between codon and anticodon</u></i></p>
      <p>Fewer than 60 tRNAs needed because of wobble pairing
      <br>but organisms generally have redundant tRNAs (Ecoli has 86)</p>
      <p>There are also tRNAs that have different anticodon loops, but are charged with the same amino acid</p>
      <p></p>

    </div>
    
    <div class='topicOverview' id='subDes23'>
          <h2>Elongation</h2>
          
        <p>Overview (add image)</p>
          
      <p><i><u>Binding of aminoacylated tRNA to the A site</u></i></p>
      <p>- Ef-Tu binds GTP, then binds aminoacylated tRNA (aa-tRNA).
      <br>- This complex then binds The ribosome with cofactors
      <br>- Amino acid is passed to growing strand, GTP gets hydrolyzed
      <br>- EF-Tu-GDP leaves, EF-Ts replaces GDP with GTP</p>
      <p>(Show image)</p>
      <p>Guanine Exchange Factor (GEF) and GTPase-activating Protein (GAP) work to acitave or inactivate proteins by binding or hydrolyzing GTP to protein</p>
      <p>Ef-Tu-GTP-aa-tRNA complex binds to the A site of the ribosome<br><b>One last chance to reject the amino acid before it's incorported</b>
      <br>Irreversible GTP hydrolysis step allows for an additional "proofreading" step</p>
      <p><b>Conformational changes of decoding center</b>
      <br>- A1492 and A1493 form the A-minor interactions with the anticodon mRNA base pairs.
      <br>- First discovered because antibiotic paromycin induces "flipping out" of these bases even in the absence of mRNA or tRNA.</p>
      <p>- Contacts are formed between A1493 and the ribose of each nucleotide in the codon-anticodon base pair
      <br>- These interactions are not formed with the thrid base of the codon, allowing degeneracy by enabling wobble conformations</p>
      
      <p><i><u>Formation of a new peptide bond</u></i></p>
      <p><b>Puromycin: an antibiotic and valuable experimental tool</b> <br>Puromycin becomes an acceptor for the growing polypeptide on a ribosome. Polypeptide gets passed to Puromycin, translation stops, ribosome disassembles
      <br><b>Puromycin reaction is great for assaying peptide bond formation</b></p>
      <p>Peptidyl-Transferase activity is mostly kept after deproteinizing the ribosome.
      <br>- Used Puromycin assay and thin-layer chromatography
      <br>Significant activity detected for ribosomes or 50S subunits even with more than 90% of protein removed
      <br>Strongly suggests RNA is catalytic part of ribosome (ribozyme)</p>
      <p><b>50S structure shows no proteins near active site for peptidyl transfer</b>
      <br>Proteins snake towards it, but not into it</p>
      <p><b>The Chemistry of Peptidyl Transfer</b>
      <br>- No specifically bound Mg2+ ions observed in reaction center
      <br>- No individual nucleotide substitutions within 23S rRNA give large effects
      <br>- Deletion of 2' OH of P site tRNA effectively abolishes catalysis.</p>
      
      <p><i><u>Translocation of the peptidyl-tRNA to the P-site</u></i></p>
      <p><b>Translocation assayed by release of deacetylated tRNA or by puromycin assay</b>
      <br>Initially concluded that GTP binding, but not hydrolysis, was necessary for translocation</p>
      <p><b>Experiment:</b> Signal for translocation from a fluorescent peptidyl-tRNA in the A site
      <br>- GTP hydrolysis monitored separately with radiolabeled GTP
      <br>- GTP is hydrolyzed faster than translocation occurs, so hydrolysis precedes translocation
      <br>- But translocation can still occur, albeit more slowly, without GTP hydrolysis
      <br>- much of translocation is complex and not well understood</p>
      <p><b>Ribosome Profiling: A genome-wide means of measuring translation</b>
      <br>- Translation is stopped (freezing or antibiotic)
      <br>- Translating mRNAs are isolated
      <br>- Surrounding RNA is degraded, leaving only RNA protected by Ribosomes
      <br>- A library is generated and sequenced
      <br>Allows determination of which genes are being translated, and also can locate pause sites by density of ribosomes within a given mRNA</p>
      <p>Comparison with mRNA abundance allows determination of translation efficiency
      <br>(Show pic)</p>

    </div>
    <div class='topicOverview' id='subDes33'>
          <h2>Termination</h2>
      <p><i><u>Early nonsense mutation/supressor strains</u></i></p>
      <p>Put in an amber stop codon in the middle of a gene
      <br>Expressed in a Amber suppressor strain to regain activity of following gene (in an operon)</p>
      
      <p><i><u>Mechanism of Suppression</u></i></p>
      <p>Mutation in tRNA allows recognition of stop codon and incorporation of amino acid</p>
      <p>More than one tRNA can be mutated to become a suppressor</p>
      
      <p><i><u>Release Factors</u></i></p>
      <p><b>A simple assay for release factors</b>
      <br>- fMET-tRNA loaded in P site bound to AUG
      <br>- Tri-nucleotide corresponding to stop codon added
      <br>- Addition of ribosome fractions containing appropriate release factor would allow release of labeled fMET into solution</p>
      <p><b>Bound Release Factor I is positioned for hydrolysis of Aminoacyl Linkage</b>
      <br>- Conserved GGQ motif is positioned near peptidyl transferase center</p>
    </div>
    <div class='topicOverview' id='subDes43'>
          <h2>Quality Control</h2>
      <p><i><u>Bacterial tmRNA</u></i></p>
      <p>Some aberrant RNAs lack termination codons, causing ribosomes to read to the end and then stall</p>
      <p>tmRNA provides three "services" to the cell:
      <br>- It allows recycling of stalled ribosomes
      <br>- It targets the aberrant RNA for degradation
      <br>- It targets the aberrant peptide for degradation</p>
      <p><b>Mechanism:</b>
      tmRNA goes in P-site,
      <br>SmpB helps transfer peptide chain to tmRNA
      <br>Ribosome reads mRNA portion of tmRNA, produces C-term of protein
      <br>Protein released, BUT WAIT, tmRNA encoded for degradation signal on C-term of protein! BOOOOM!!! Time bomb</p>
      
      <p><i><u>Nonstop decay: when eukaryotic mRNAs lack a stop codon</u></i></p>
      <p>Analogous to tmRNA process, except uses Ski7p protein that recruits an exosome</p>
      <p>Can occur when (1) Polyadenylation occurs prematurely, (2)When transcription aborts (3) Upon imcomplete 3'-5' decay</p>
      <p><b>Decay of mRNAs with premature termination codons (PTC)</b>
      <br>- NMD is dependent on translation and reading frame, indicating that the ribosome identifies the stop codon</p>
      <p> Mammals use splice history to determine when a stop codon is premature
      <br><b>Exon Junction Complex</b> (EJC) is deposited upon splicing
      <br>- If a stop codon is far upstream from the last EJC, the stop codon is labeled as premature and NMD is triggered</p>
      
      <p><b>mRNA Decay Pathway in NMD</b>:
      <br>- Principally decapping and 5'-3' exonuclease both in yeast and mammals
      <br>Here, decapping does not require deadenylation
      <br>Dcp1/2 recruited by Upf1p</p>
      
      <p><i><u>No-go mRNA Decay: When Translation Stalls</u></i></p>
      <p> Requires protein Dom34, related to release factor
      <br>- Binding of DOM34 and Hbs1 presumably recruit an endonuclease
      <br>- Decay then proceeds 3'-5' by the exosome (for the 5' fragment) and 5'-3' by Xrn1p (for the 3' fragment)</p>
      <p>These are the eukaryotic mRNA decay pathways we talked about earlier</p>
      <p></p>

    </div>
    
  </div> <!--relative box-->
    
     <div class='topicHeader'>
      <h1>Ribosome Biogenesis</h1>
    </div>
    <div class='topicDescription'>
      <p>* ... TBD</p>
      <p>* </p>
      <p>* </p>
    </div>
    
  <div id='relativeBox4' class='relativeBoxes'>
    <div id='subheadingScroll4' class='scrollFixBar goBack1'>
      <div id='sub14' class='topics'>
        <h2>In vitro Assembly</h2>
      </div
      ><div id='sub24' class='topics'>
        <h2>rRNA Modification</h2>
      </div
      ><div id='sub34' class='topics'>
        <h2>U3 Folds Laundry</h2>
      </div
      ><div id='sub44' class='topics'>
        <h2>Ribosome Proofreading</h2>
      </div>

    </div>
    
    <div class='topicOverview' style='margin-top:15vh' id='subDes14'>
      <p>Bacteria have a 70S ribosome (50S and 30S), while eukaryotes have an 80S ribosome (60S and 40S)</p>
      <p>All of the information necessary for assembly is encoded within the rRNA and proteins themselves</p>
          <h2>In vitro Assembly</h2>
      <p><i><u>Hierarchal assembly</u></i></p>
      <p>16S rRNA and 21 ribosomal proteins form an active 30S subunit</p>
      <p><b>In vitro assembly of bacterial ribosomes is hierarchial</b>
      <br>- Primary prinding proteins bind first and are prerequisite for the binding of secondary and tertiary proteins
      <br>- Individual domains can fold independently
      <br>- Primary binding proteins are domain-specific<br>- Assembly of tertiary proteins is dependent on previous assembly of multiple domains</p>
      <p><b>Hierarchy of Assembly in vitro follows the temporal order of transcription and emergence of nacent RNA from the RNA polymerase</b></p>
      
      <p><i><u>Kinetic Analysis of 30S assembly</u></i></p>
      <p><b>Pulse-Chase - MS experiment:</b> Add RNA and N15 labeled proteins
      <br>- after set amount of time, chase with normal N14 protiens until 30S is assembled
      <br>-Take MS of proteins and identify when labeled protein is incorporated</p>
      <p>Kitetic analysis gives early assembly maps
      <br>- Early assembly relys on independent RNA folding with proteins stabilizing RNA structures
      <br>- Later events are more interdependent
      <br>Such hierarchial assembly helps ensure that ribosomes are assembled correctly before they can be used in translation</p>
      <p><b>Order of spontaneous assembly in vitro follows the temporal order of transcription and emergence of the nacent RNA from RNAP</b></p>
      
      <p><i><u>Assembly is more efficient in vivo</u></i></p>
      <p>Because:
      <br>- Co-transcriptional assembly in vivo may help avoid kinetic traps that often arise in large RNAs
      <br>- RNA processing of precursor 16S rRNA may enhance the rate of assembly
      <br>- 30S subunit proteins bind their RNA faster in the presence of protein cofactors</p>
      <p>Additional factors assist in the assembly of the ribosomal subunits in vivo in bacterial cells
      <br>- surprisingly, most of these are dispensable for viability</p>
      
      <p><i><u>Eukaryotic Ribosome Assembly is more Complex</u></i></p>
      <p>Ribosomes are assembled co-transcriptionally on the pre-rRNA transcript in the nucleolus</p>
      <p>There are more than 200 assembly factors in yeast and more than 300 in human cells. Most are essential.</p>
      <p><b>Why Eukaryotic Assembly more Complex?:</b>
      <br>- More complex folding pathways, ore integration of metabolic pathways?
      <br>- Some factors prevent the precocious interaction of translation factors
      <br>- Some factors are needed for removing other factors
      <br>- Additional levels of quality control</p>
      <p><b>Numbers:</b>
      <br>- In yeast, 60% of translation is devoted to rRNA. 1/2 of transcription and 90% of splicing devoted to making ribosomal proteins
      <br>- the ratio of RNA to DNA in rapidly dividing yeast is 50:1 (80% rRNA, 15% tRNA, 5% mRNA)
      <br>- Growing yeast produce 2,000 ribosomes per minute</p>
    </div>
    
    
    <div class='topicOverview' id='subDes24'>
          <h2>rRNA Modification</h2>
      <p><i><u>Modification Overview</u></i></p>
      <p>The organization of pre-rRNA is conserved between bacteria and eukaryotes (like the gene structure)
      <br>- Eukaryotes have and <b>Internal transcribed spacer</b></p>
      <p><b>Bacterial</b> pre-rRNA is cut a bunch by different specific RNases and have modifications
      <br><b>Eukaryotic</b> rRNAs are also cut by specific RNases and an exosome. Also processed by different enzymes, but this process also produces an Internal Transcribed Spacer 1</p>
      <p>Ribosomal RNAs are highly modified with <b>pseudouridylation and 2'-O-methylation</b></p>
      
      <p><i><u>snoRNAs</u></i></p>
      <p>Most rRNA modifications of eukaryotic rRNA are <b>guided by small nucleolar (sno) RNAs</b>
      <br>- Box "H/ACA" snoRNAs direct pseudouridylation
      <br>- Box "C/D" snoRNAs direct 2'-O methylation</p>
      <p>Eukaryotes encode many snoRNAs (over 100 in humans)</p>
      <p>Modification sites <b>cluster around functionally important centers</b> and are conserved between bacteria and eukaryotes</p>
      <p>snoRNAs <b>can be engineered to recode mRNAs</b></p>
    </div>
    <div class='topicOverview' id='subDes34'>
          <h2>U3 Folds Laundry</h2>
      <p><i><u>U3 snoRNA</u></i></p>
      <p>The C/D box U3 snoRNA does not direct 2'-O methylation: it is an <b>RNA chaparone</b> that orchestrates RNA folding and processing</p>
      <p>U3 snoRNA binds multiple sites in pre-rRNA to <b>promote pre-rRNA folding</b></p>
      <p>U3 snoRNA brings together the RNA elements of the central pseudoknot</p>
      
      <p><i><u>Dhr1 helicase removes U3 snoRNA</u></i></p>
      <p>U3 snoRNA must be removed to allow the central pseudoknow to fold. Dhr1 helicase removes U3 snoRNA to complete folding</p>
      <p>RNA conformation changes, and RNP rearrangements are driven by RNA helicases.
      <br>- 19 helicase-family proteins in yeast work in ribosome assembly.
      <br>- 17 are essential</p>
      <p><b>Ribosomes are largely pre-assembled in the nucleolus, but undergo maturation in the cytoplasm</b></p>

    </div>
    <div class='topicOverview' id='subDes44'>
          <h2>Ribosome Proofreading</h2>
      <p><i><u>Trans-acting Factors</u></i></p>
      <p>Nacent ribosomal subunits <b>entering the cytoplasm are inactve</b>
      <br>- Transacting factors block their function
      <br>- final maturation is coupled to functional proofreading</p>
      <p><b>Transacting factors occupy the binding sites for translation initiation factors to prevent immature subunits from engaging in translation</b></p>
      <p>Final processing of 18S occurs in the cytoplasm
      <br>Further cleavage by specific RNases
      <br>- Internal Transcribed Spacer 1 (ITS1) is also cleaved in cytoplasm</p>
      <p>ITS1 is proposed to lay in the mRNA channel</p>
      
      <p><i><u>eIF5B recruits the 60S subunit during Translation Initiation</u></i></p>
      <p><b>During Translation initiatino, eIF5B recruits the 60S subunit</b></p>
      <p>During 40S maturation, eIF5B recruits 60S to promote the maturation of the 40S subunit
      <br>- Nob1 (cleavage factor) can now cleave (and mature) ITS1</p>
      <p><b>Functional proofreading of both subunits involves translation-like events</b>
      <br>- The final steps of 40S maturation involve joining with the 60S subunit in a translation initiation-like event
      <br>- The final steps of 60S maturation require a homolog of EF-G and Ribosome Recycling Factor in a translocation-like event</p>
      
      <p><i><u>Summary + extras</u></i></p>
      <p><b>Correct Assembly of Ribosomes is Ensured By:</b>
      <br>- Hierarchical assembly (structural proofreading)
      <br>- Translation-like events, initiation for the small subunit, translocation for the large subunit (Functional Proofreading)</p>
      <p>Ribosomopathies: Diseases that arise from defective ribosomes:
      <br>- Most of these diseases arise from dysfunctional ribosomes that have largely been assembled correctly, and escape late quality control checks.</p>
    </div>
    
  </div> <!--relative box-->
    
     <div class='topicHeader'>
      <h1>Proteomics</h1>
    </div>
    <div class='topicDescription'>
      <p>* </p>
      <p>* </p>
      <p>* </p>
    </div>
    
  <div id='relativeBox5' class='relativeBoxes'>
    <div id='subheadingScroll5' class='scrollFixBar goBack1'>
      <div id='sub15' class='topics'>
        <h2>Sample Prep % MS</h2>
      </div
      ><div id='sub25' class='topics'>
        <h2>Quantification</h2>
      </div
      ><div id='sub35' class='topics'>
        <h2>Other Protein Methods</h2>
      </div>

    </div>
    
    <div class='topicOverview' style='margin-top:15vh' id='subDes15'>
      <h2></h2>
          <h2>Sample Prep & MS</h2>
      <p><i><u>Overview: Proteomics</u></i></p>
      <p><b>Can Measure:</b>
      <br>- Absolute and Relative Abundance
      <br>- Turnover
      <br>- PTM Analysis
      <br>- Protein Isoforms
      <br>- Protein-Protein Complexes and Interactions
      <br>- Localization</p>
      <p>Apply all of this generated data to <b>Computational analysis tools</b></p>
      <p><b>The Proteome is Complex:</b> Complexity increases as you go from genes, through transcription, translation, post-translational modification ...</p>
      <p><b>Workflow:</b>
      <br>- Purify complexes, isolate membranes/organelles
      <br>- Solubilize, fractionation
      <br>- Run SDS-PAGE gel, extract protein
      <br>- Digest proteins to get peptides
      <br>- Run through MS -> Data mining
      <br>- Target selection and validation</p>
      
      <p><i><u>Proteomics Sample Prep</u></i></p>
      <p><b>Mass spec requires buffer-free samples:</b>
      <br>-Run a gel, use in-gel-digest to remove unwanted buffers
      <br>- TCA or acetone precipitate, wash lysate
      <br>- Membrane centrifuge, dialysis to remove salts
      <br>- Avoid use of polymers and detergents. Use urea and MS-friendly detergents
      <br>- Separate and purify components -HPLC</p>
      <p>After sample purification (remove salts), Proteins must be fragmented and cleaned for use in MS</p>
      <p></p>
      
      <p><i><u>Mass Spec</u></i></p>
      <p>Mass spec breaks up fragmented peptides and identifies composition of tiny peptide fragments</p>
      <p>Fragment sequences are then paired together to build full sequences</p>
      
      <p><i><u>LC-MS/MS Protein Identification</u></i></p>
      <p><b>LC-MS/MS</b>
      <br>- Proteins are fragmented into peptides
      <br>- <b>Peptides are then separated</b> by Liquid Chromatagraphy
      <br>- Fragments are analyzed by Tandem MS (MS/MS)</p>
      <p></p>
      <p></p>
    </div>
    
    <div class='topicOverview' id='subDes25'>
          <h2>Quantification</h2>
      <p><i><u>False Discovery Rate</u></i></p>
      <p>False Discovery Rate calculated by searching the data with a decoy DB to provide statistical confidence measure for peptide identifications</p>
      
      <p><i><u>Isobaric Tagging: iTRAG/TMT</u></i></p>
      <p>not sure, look at pic?</p>
      <p>Can also use "Skyline" for Targeted Quantitiation</p>
      <p>Can quantify with synthetic peptides as controls</p>
      
      <p><i><u>Detecting Protein Modifications by MS</u></i></p>
      <p>Start with microgram levels of single protein or mg of complex sample
      <br>- Use modification enrichment: affinity chromatography, antibody pulldown, biotinylation, click chemistry
      <br>- Purify protein/protein complex
      <br>- Try targeted MS/MS on modified peptide
      <br>- Use Ascore to assess site localization
      <br>- Validate with synthetic modified peptide standard or antibody</p>
      <p><b>Co-immunoprecipitation:</b>
      <br>- Fuse antibody to beads (biotin streptavidin)
      <br>- lyse cell and let proteins bind to antibody
      <br>- isolate antibody/protein, wash protein off
      <br>- Can use many protein tags on target protein to have antibody recognize it (HIS, FLAG, V5)</p>

    </div>
    
    <div class='topicOverview' id='subDes35'>
          <h2>Other Protein Methods</h2>
      <p><i><u>GST in-vitro Pulldown</u></i></p>
      <p>Bind affinity ligand to bead
      <br>- Bind "bait" Protein (that makes interaction with target protein) to the affinity ligand
      <br>- wash. Then bind "prey" target protein
      <br>- wash. then elute bait-prey complex and MS</p>
      
      <p><i><u>Crosslinking MS</u></i></p>
      <p>Crosslink protein complex
      <br>- proteolysis -> read by LC-MS/MS
      <br>Map of cross-links set distance restraints</p>
      
      <p><i><u>Hydrogen Deuterium Exchange</u></i></p>
      <p>Isolate protein target
      <br>- Incubate in D2O, <b>Amide hydrogens in disordered regions exchange fastest</b>
      <br>- Freeze, proteolysis
      <br>- Read by LC-MS/MS
      <br><b>Identify disordered regions of proteins</b></p>
      
      <p><i><u>Can use different types of affinity binders to isolate proteins/complexes</u></i></p>
      <p>Antibodies
      <br>-Recombinant antibody fragments
      <br>- Protein scaffolds
      <br>- DNA or RNA aptamers
      <br>- Small binding peptides or compounds</p>
    </div>
    
  </div> <!--relative box-->
  
       <div class='topicHeader'>
      <h1>Regulatory Small RNAs</h1>
    </div>
    <div class='topicDescription'>
      <p>* Small RNAs, in the forms of siRNAs and miRNAs, play large roles in the regulation of gene expression in eukaryotes. They are important in normal cell metabolism, development, and defense against invaders.</p>
      <p>* siRNAs are produced from longer segments of dsRNA by Dicer, assembed into RISC, and targeted to mRNAs with perfect complementarity, giving silencing by cleavage and degradation of the RNA or by formation of heterochromatin</p>
      <p>* The pathway for miRNA has many steps in common with that for siRNA. However, miRNAs are processed from hairpin precursors, and they most often have imperfect complementarity with their targets, giving effects on translation and on mRNA decay by standard pathways rather than Ago-mediated cleavage of the mRNA</p>
    </div>
    
  <div id='relativeBox6' class='relativeBoxes'>
    <div id='subheadingScroll6' class='scrollFixBar goBack1'>
      <div id='sub16' class='topics'>
        <h2>RNAi Components</h2>
      </div
      ><div id='sub26' class='topics'>
        <h2>RNAi Mechanisms</h2>
      </div
      ><div id='sub36' class='topics'>
        <h2>miRNA & its Mechanism</h2>
      </div>

    </div>
    
    <div class='topicOverview' id='subDes16'>
    <br><br><br><br><br>
    <p><b>Overview of siRNA and miRNA:</b>
    <br>- dsRNA is formed (serves as a trigger)
    <br>- Dicer enzyme cleaves dsRNA into fragments
    <br>- Arugonaute protein complex binds with dsRNA fragments and carries out function (gene regulation)
    <br>(anneal pic)</p>
    
          <h2>RNAi Components</h2>
      <p><i><u>History</u></i></p>
      <p>Some flowers have a purple and white color. Instead of adding purple pigment, some cells produce transgenes that suppress the purple pigment production</p>
      <p><b>Injection of dsRNA in worms shown to cause destruction of specific mRNA</b>
      <br>- Injected silencing dsRNA into gonads
      <br>- dsRNA had to include exons, introns and promoter didnt work
      <br>- Effect was incredibly potent and even spead to other cells within the worm
      <br>- Termed <b>RNA interference</b>. Very useful as a tool</p>
      <p><b>dsDNA is processed to small fragments of well-defined length</b>
      <br>- Used a system from fly embryo lysate
      <br>- Added dsRNA corresponding to luciferase gene
      <br>- Fragments were the same length regardless of which strand was labeled
      <br>- production of fragments did not require the luciferase mRNA, suggesting that the processing is upsteam of effect on mRNA</p>
      
      <p><i><u>Producing and Cutting dsRNA</u></i></p>
      <p><b>Producing dsRNA</b>
      <br>- some dsRNAs have viral origin, but not all
      <br>- Genomic repetitive sequences also are a source of siRNA
      <br>- Some even regulate other genes</p>
      <p><b>Dicer: Cutting dsRNA into small fragments</b>
      <br>- <b>Dicer protein</b> cuts dsRNA into small fragments
      <br>- PAZ domain binds RNA end
      <br>-RNase III domains cut RNA to produce2nt 3' overhang
      <br>- Roles of other domain remain unclear</p>
      
      
      <p><i><u>dsRNA Fragment-Protein Complex</u></i></p>
      <p><b>Assembly of RISC requires additional proteins</b>
      <br>- Processing of dsRNAs into RISC requires accessory proteins: <b>R2D2</b> in flys <b>forms complex with dicer</b>
      <br>- RISC loading complex includes Dicer, R2D2 and possibly other proteins
      <br>- Ago2 cleaves the passenger strand, leading to its ejection</p>
      <p><b>Removal of Passenger Strand is Accelerated by C3PO</b>
      <br>- C3PO enhances mRNA cleavage by Ago 2 programmed with dsRNA
      <br>- Complex of two proteins, Translin and Trax, both of which are required for activity
      <br>- Trax hase RNase activity, which is required for promotion of RISC complex
      <br>- Suggests that RNase activity is used to remove passenger strand to activate RISC
      <br>- Same activity may also remove cleaved mRNA in subsequent functional cycles</p>
      <p><b>Argonaute: Central component of the RISC complex</b>
      <br>- One strand of the dsRNA produced by Dicer is retained in the RISC complex in association with argonaute
      <br>- in structure without mRNA, guide strand nucleotides 2-6 have bases exposed and available for base pairing
      <br>- One domain adopts RNase H fold and some Ago proteins can cleave the "passenger strand": aka the mRNA</p>
      

    </div>
    
    <div class='topicOverview' id='subDes26'>
          <h2>RNAi Mechanisms</h2>
      <p><i><u>Mechanism Overview</u></i></p>
      <p>siRNA signal can be amplified by dsRNA synthesis</p>
      <p>Imperfect base-pairing between guide strand and target can give non-degradative silencing (miRNA pathway)
      <br>- Off-target responses of synthetic siRNA often use this miRNA pathway</p>
      <p><b>siRNA pathway can act at the level of DNA also</b>
      <br>- Transcriptional silencing can involve methylation of histones and/or genomic DNA</p>
      
      <p><i><u>In Some organisms, siRNA signal is amplified and spread</u></i></p>
      <p>New siRNAs appear against other regions of targeted mRNAs</p>
      <p>Signal can even spread to other cells in plants, and some animals</p>
      <p>Amplification seems to be present only in certain organisms</p>
      
      
      <p><i><u>A Modular Arms Race Between Virus and Host</u></i></p>
      <p>Cell co-opts the viral RNA and uses it against itself
      <br>- System is easily adaptable to any new virus or foreign invader</p>
      <p>Virus could use RNAi to compromise host
      <br>- Why not mutate so that a viral sequence corresponds to an important host sequence?</p>
      <p>Viruses have most likely done this, and there is likely a continual arms race
      <br>- Even amplification and spread of an siRNA signal could be hijacked by viruses to 'prime' new cells for easy infection</p>
      

    </div>
    
    <div class='topicOverview' id='subDes36'>
          <h2>miRNA & its Mechanism</h2>
      <p><i><u>History</u></i></p>
      <p>GeneA was known to repress GeneB, but required the 3'UTR of GeneB to repress it</p>
      <p>Subsequenct analysis showed that GeneA produced a non-coding RNA with complementarity to the critical region in GeneB</p>
      <p>Repression in L2 developmental stage was shown to be independent of degradation of the mRNA or even changes in polysome profile
      <br>- Now known to be at least 3000 miRNAs in humans</p>
      
      <p><i><u>miRNA Biogenesis and RISC assembly</u></i></p>
      <p>Transcription is by RNA pol II, and transcripts are capped and polyadenylated
      <br>- Processed from an imperfect hairpin within transcript, called <b>pri-miRNA</b></p>
      <p>In plants, both processing steps to produce short dsRNA are by Dcl1 in nucleus
      <br>- In <b>animals</b>, hairpin is excised by Drosha or by splicing in nucleus to produce pre-miRNA</p>
      <p>Then Dicer performs final processing step in the cytoplasm</p>
      <p><b>Extent of Complementarity Determines Whether miRNA Gives Cleavage or Repression of Target mRNA</b>
      <br>- Whither perfect or near-perfect complmentarity, miRNA can act like siRNA, giving cleavage of target
      <br>- This pathway operates extensively in plants, less so in animals</p>
      
      
      <p><i><u>Possible Mechanisms of Translation Inhibition by miRNA</u></i></p>
      <p><b>Translation Initiation or Elongation Block</b></p>
      <br>
      <p><b>Experiment:</b> Does Translation repression involve competition for cap binding?
      <br>-Mouse cell-free extract
      <br>- Luciferase mRNA with miRNA binding sites for endogenous let-7 miRNA
      <br>- Decreased 80S peak by glycerol gradient centrifugation, rescued by comp. oligo to let-7
      <br>- Decreased inferred to reflect decreased translation initiation
      <br>- Translation also inhibited, and inhibition eliminated with inclusion of IRES in mRNA
      <br>- Inhibitory effect on translation reversed by increased eIF4F concentration</p>
      <p>Another possible mechanism is <b>mRNA Degradation</b>:
      <br><b>Experiment:</b>
      <br>- mRNA decay followed in fly cells
      <br>- Actinomycin D used to block transcription, reporter mRNA level then followed by Northern blot
      <br>- Decay of mRNA dependent on GW182, Ago, and deadenylation and decapping machineries</p>
      <p><b>MiRNA effects are Mediated Through GW182 Protein</b>
      <br>- GW182 shown to be required for miRNA effects
      <br>- Also sufficient: tethering GW182 without Ago gave silencing
      <br>- GW182 shown to bind PolyA-binding protein(PABP)</p>
      
      <p><b><i>Possible Mechanism of miRNA-induced Effects on Translation and Stability</i></b></p>
      <p>Silencing may arise from loss of translation as well as induced degradation</p>
      <p>Degradation uses general pathways, prominently decapping followed by 5'-3' decay</p>
      <p><b>Biological Roles of miRNA-induced Gene Repression</b>
      <br>- Many miRNAs interact with multiple mRNA targets
      <br>- mRNA untranslated sequences may evolve in context of milieu of miRNAs that allow fine-tuning of expression levels
      <br>- Tuning effect is the same whether miRNA is constituitively expressed or switched on
      <br>- Small effect can become large if transcription is shut off</p>
      

    </div>
    
  </div> <!--relative box-->
  
       <div class='topicHeader'>
      <h1>lincRNAs</h1>
    </div>
    <div class='topicDescription'>
      <p>* Chromosomes are arranged to allow long-range contacts between regulatory elements and promoters, gene ends, and much longer-range contacts involving multiple genes. Contacts maps have led to the view that chromosomes are arranged in a segmented way.</p>
      <p>* Genomic methods have indicated the presence of large numbers of long, intervening non-coding RNAs (lincRNAs). While many of these RNAs are likely to be non-functional, some are known to be functional and many others are likely to be functional as well.</p>
      <p>* Some lncRNAs act in cis on DNA adjacent to their transcription sites, while others function in trans to regulate gene expression, functioning in the nucleus or the cytoplasm.</p>
      <p>* More functions almost certainly exist and are yet to be discovered.</p>
    </div>
    
  <div id='relativeBox7' class='relativeBoxes'>
    <div id='subheadingScroll7' class='scrollFixBar goBack1'>
      <div id='sub17' class='topics'>
        <h2>Organization of Chromatin</h2>
      </div
      ><div id='sub27' class='topics'>
        <h2>lincRNA Identification</h2>
      </div
      ><div id='sub37' class='topics'>
        <h2>lincRNA Functions</h2>
      </div>

    </div>
    
    <div class='topicOverview' id='subDes17'>
      <br><br><br><br><br><br><br>
          <h2>Organization of Chromatin</h2>
      <p><i><u>Long-range genomic interactions & their detection</u></i></p>
      <p>Sequences can <b>Loop</b> to form interactions with sequences far up/downstream
      <br>- Loops vary in length and functions:
      <br>- Regulation of gene expression
      <br>- Enhancement of Transcription</p>
      <p><b>Experiment: Chromosome Conformation Capture (3C):</b>
      <br>- Uses random crosslinking followed by restriction digestion and ligation
      <br>- Interactions of yeast DEA mapped by PCR with primer pairs across chromosome (image)</p>
      <p><b>DNA loops between enhancers and promoters contain mediator and cohesin</b>
      <br>- shRNA screen for transcription regulator Oct4 and maintenance of embryonic cell state
      <br>- Knockdown of mediator or cohesin complex components reduced Oct4 levels and changes in cell morphology
      <br>- Co-IP showed association of mediator and cohesin
      <br>- ChIP-Seq and 3C showed interaction of enhancer and promoter regions in mES cells (stem cells) but not MEF (fibroblasts)
      <br><br>- Suggests model in which cohesin forms ring to connect upstream and downstream DNA regions</p>
      
      <p><i><u>Mapping of Long-range Genomic Interactions</u></i></p>
      <p><b>Experimental Method: Hi-C</b>
      <br>- Provides genome-wide map of interactions
      <br>- Interaction pattern shows extensive long-range contacts</p>
      <p>Contact probability decreases with distance
      <br>- Intrachromosomal contacts are more likely than contacts between chromosomes
      <br>- <b>Power dependence suggests fractal, non-random folding</b></p>
      
      
      <p><i><u>Transcriptional Factories</u></i></p>
      <p><b>Transcription is localized in Discrete 'Transcription Factories'</b>
        <br>- Transcripton sites detected by bromo-UTP and antibody
      <br>- Approximately 10,000 factories per cell, but more than 65,000 active molecules of RNAP II
      <br>- Presumably regions of transcriptioin factor density, possibly multiple genes</p>
      <p>Other processes are also localized in regions:
      <br>-Transcription
      <br>- Replication
      <br>- Repair</p>
      <p></p>
      

    </div>
    
    <div class='topicOverview' id='subDes27'>
          <h2>lincRNA Identification</h2>
      <p><i><u>Possible Functions & Identification</u></i></p>
      <p><b>What are possible functions of lincRNAs?</b>
      <br>- Functionless, but tolerated by nature: (not highly conserved, expressed at low levels, would still be capped and polyadenylated
      <br>- Act of transcription is functional, but RNA product is not functional: (ex- for activation of a neighboring gene)
      <br>- Roles of RNA product in cis on DNA or in trans, nuclear or cytoplasmic</p>
      <p><b>Identification of Candidate lincRNAs</b>
      <br>- Several methods have been used to identify putative lincRNAs
      <br>- Histone modification pattern
      <br>- cDNA with cap purification
      <br>- next-gen sequencing for polyA position
      <br>- "Expressed Sequence Tag" corresponds to cDNA</p>
      
      <p><i><u>Common Criteria for lincRNAs</u></i></p>
      <p>No single criterion is sufficient, but a combination of a few is useful</p>
      <p>Some criteria include:
      <br>- Data for transcript reconstruction
      <br>- Genomic features and filters
      <br>- Coding-potential filters</p>
      
      
      <p><i><u>Issues with Classifying Coding and Non-coding RNAs</u></i></p>
      <p>Random sequences are reasonably likely to have ORF sequences</p>
      <p>Even ribosome association may not indicate functional translation</p>
      <p><b>Possible roles of Ribosome Association</b>
      <br>- Translation might protect proteins that bind the RNA downstream
      <br>- Translation might function by tethering ribosomes
      <br>- Translation might promote decay of the RNA</p>
      

    </div>
    
    <div class='topicOverview' id='subDes37'>
          <h2>lincRNA Functions</h2>
      <p><i><u>Subcellular Localization of lincRNAs</u></i></p>
      <p>Inaggregate, lincRNAs are localized to the nucleus to a greater extent than mRNAs</p>
      <p>However, individual lincRNAs can be predominantly nuclear or predominanly cytoplasmic</p>
      
      <p><i><u>Possible Functions of lincRNAs</u></i></p>
      <p><b>X-inactivation by Xist RNA</b>
      <br>- Xist lincRNA can bind to an X chromosome and recruit PRC2 complexes, which I assume inactivates the chromosome</p>
      <p><b>lincRNAs and Nuclear Organization</b>
      <br>- Connect two regions by interacting with enhancers and promoters
      <br>- Gene to gene or chromosome to chromosome gene expression regulation
      <br>- Barr body formation
      <br>- Form nuclear speckles, paraspeckles, and in nucleolus for some reason</p>
      <p><b>lincRNAs Interactions with Chromatin</b>
      <br>- <b>ChIRP-seq</b> method used to identify interactions of lincRNAs
      <br>- Method is analogous to ChIP-seq, but purification is of lincRNA
      <br>- Binding regions mapped for lincRNA HOTAIR, revealed co-occupancy with H3K27me3 modifications</p>
      <p><b>A circular RNA acts as a Sponge for an miRNA</b>
      <br>- RNA circularizes by "back splicing"
      <br>- Includes roughly 70 binding sites for miR-7, which titrates it away from its mRNA targets</p>
      <p>lincRNAs can also target an mRNA for degradation by pairing in the 3'UTR, activating a degradation pathway distinct from RNAi</p>
      

    </div>
    
  </div> <!--relative box-->
    
      
  </body>

</html>